tools/topology/platform/intel/intel-generic-dmic.m4

```diff
@@ -0,0 +1,98 @@
+#
+# Topology for digital microphones array
+#
+
+# Include topology builder
+include(`utils.m4')
+include(`dai.m4')
+include(`pipeline.m4')
+
+# Include TLV library
+include(`common/tlv.m4')
+
+# Include Token library
+include(`sof/tokens.m4')
+
+include(`platform/intel/bxt.m4')
```

Comment:
including this means this file is not generic. In fact most of these includes are not needed (since this file will always be included into a mchine file).

Reply:
I will remove these headers